src/flamenco/runtime/tests/fd_exec_instr_test.c

```diff
@@ -263,7 +263,7 @@ fd_exec_test_instr_context_create( fd_exec_instr_test_runner_t *        runner,
 
   /* Initial variables */
   txn_ctx->loaded_accounts_data_size_limit = FD_VM_LOADED_ACCOUNTS_DATA_SIZE_LIMIT;
-  txn_ctx->heap_size                       = fd_ulong_max( txn_ctx->heap_size, FD_VM_HEAP_SIZE ); /* FIXME: bound this to FD_VM_HEAP_MAX?*/
+  txn_ctx->heap_size                       = FD_VM_HEAP_MAX;
```

Comment:
I think it should be:
```
fd_ulong_min( 
  fd_ulong_max( txn_ctx->heap_size, FD_VM_HEAP_DEFAULT ),
  FD_VM_HEAP_MAX )
```
i.e. by default an ix has 32kB of heap (DEFAULT). If the ix was set with extra heap, we use that. At most 256kB heap (MAX)?

(the other change below looks correct)

Reply:
`txn_ctx->heap_size` isn't initialized, looking at the code above. So the suggestion would return an arbitrary value that we cannot control. This can be a problem with differential fuzzing.

I do see the point that `FD_VM_HEAP_DEFAULT` should be, well, the default value. So I'll change it to that. This line of code has gone through a full circle 🙃

Reply:
We do have a separate mechanism for controlling heap sizes in cases where we are interested in doing so (i.e., VM fuzzers), so I think setting it to default here is fine.